lib/pbench/test/unit/server/auth/test_auth.py

```diff
@@ -378,9 +378,7 @@ def test_token_introspect_exp(self, monkeypatch, rsa_keys):
 
         with pytest.raises(OpenIDTokenInvalid) as exc:
             oidc_client.token_introspect(token)
-        assert (
-            str(exc.value.__cause__) == "Signature has expired"
-        ), f"{exc.value.__cause__}"
+        assert exc.value.exc_type == jwt.ExpiredSignatureError, f"{exc.value.exc_type}"
```

Comment:
`assert isinstance(exc.value.__cause__, jwt.ExpiredSignatureError)` would allow you to keep using the `raise [] from e`.

And in any case, the comparison should be `is` rather than `==`.